portal/src/util/eip681.ts

```diff
@@ -21,9 +26,8 @@ export function parseEIP681(url: string): EIP681 {
   }
 
   const address = addressURI[0];
-  try {
-    parseInt(address, 16);
-  } catch (_: unknown) {
+  const regex = new RegExp(ETHEREUM_ADDRESS_REGEX);
```

Comment:
ETHEREUM_ADDRESS_REGEX itself is an instance of RegExp already. No need to instantiate a new one every time with `new RegExp`.